### PR TITLE
server 3.3.0 docs release branch

### PR DESCRIPTION
**Pull request by rosieyohannan**
https://docs.google.com/document/d/1nYZyE4neGBbE9YZjMTySWEf97DtclSRdX8wGyCijDXs/edit

This PR contains all docs updates for the server 3.3 release. Just before release I will copy in the changelog items from https://github.com/circleci/circleci.com/pull/5338, when it's finalised, and I'll add in a PDF of the updated ops and installation guides.

### CODE REVIEW

**Review by devinmarieb — APPROVED** [review]
🙌

File: jekyll/_cci2/server-3-install-build-services.adoc [hunk 1]
```diff
@@ -150,8 +218,67 @@ terraform apply
 
 After Terraform is done spinning up the Nomad client(s), it will output the certificates and key needed for configuring the Nomad control plane in CircleCI server. Make sure to copy them somewhere safe.
 
+==== Nomad Autoscaler (Optional)
```
Comment (rosieyohannan) [inline jekyll/_cci2/server-3-install-build-services.adoc:221]:
```suggestion
==== Nomad Autoscaler
```

File: jekyll/_cci2/server-3-install.adoc [hunk 2]
```diff
@@ -123,32 +123,47 @@ You should have created your S3 Compatible bucket and optional IAM account durin
...
 You should have created your Google Cloud Storage bucket and service account during the prerequisite steps. 
 
 * *Storage Bucket Name (required)* - 
 The bucket used for server.
 
+* *Storage Object Expiry (required)* - 
+Number of days to retain your test results and artifacts. Set to 0 to disable and retain objects indefinitely.
+
+====== Authentication
+One of the following is required. Either select IAM keys and provide:
```
Comment (rosieyohannan) [inline jekyll/_cci2/server-3-install.adoc:159]:
```suggestion
```

File: jekyll/_cci2/server-3-install.adoc [hunk 3]
```diff
@@ -123,32 +123,47 @@ You should have created your S3 Compatible bucket and optional IAM account durin
...
+
 * *Service Account JSON (required)* - 
 A JSON format key of the Service Account to use for bucket access.
 
-* *Storage Object Expiry (optional)* - 
-Number of days to retain your test results and artifacts. Set to 0 to disable and retain objects indefinitely.
+Or select IAM role and provide:
+
+* *Service Account File (required)* - Service account file for the https://cloud.google.com/kubernetes-engine/docs/how-to/workload-identity[Workload Identity].
```
Comment (rosieyohannan) [inline jekyll/_cci2/server-3-install.adoc:167]:
```suggestion

```

**Review by ryanwohara — APPROVED** [review]

**Review by sarameisburger — APPROVED** [review]

File: jekyll/_cci2/server-3-whats-new.adoc [hunk 4]
```diff
@@ -20,6 +20,32 @@ patch releases and quarterly feature releases.
 
 toc::[]
 
+== Release 3.3.0
+
+=== New features
+
+* Nomad Autoscaler can now be used to scale Nomad clients. For more information see the https://circleci.com/docs/2.0/server-3-install-build-services/nomad-autoscaler[execution environments installation docs].
+* https://circleci.com/docs/2.0/webhooks/[Webhooks] are now available.
```
Comment (rosieyohannan) [inline jekyll/_cci2/server-3-whats-new.adoc:28]:
```suggestion
* https://circleci.com/docs/2.0/webhooks/[Webhooks] are now available.
* The Insights dashboard is now available.
```

**Pull request merged**